ci/docker/Dockerfile.build.ubuntu

```diff
@@ -127,6 +127,11 @@ RUN /work/ubuntu_julia.sh
 COPY install/ubuntu_perl.sh /work/
 RUN /work/ubuntu_perl.sh
 
+# MXNetJS nightly needs emscripten for wasm
+COPY install/ubuntu_emscripten.sh /work/
+RUN /work/ubuntu_emscripten.sh
+ENV LLVM=/work/deps/emscripten-fastcomp/build/bin
```

Comment:
I don't think you need this line? It's handled at https://github.com/apache/incubator-mxnet/blob/4774e58f37ed65a3ca97e64280230974ba4cde7e/ci/docker/runtime_functions.sh#L1468